docs/modules/extend/examples/pdf-converter-article-title-with-author-and-date.rb

```diff
@@ -0,0 +1,19 @@
+class PDFConverterArticleTitleWithAuthorAndDate < (Asciidoctor::Converter.for 'pdf')
+  register_for 'pdf'
+
+  def ink_general_heading doc, title, opts
+    return super unless opts[:role] == :doctitle # <1>
+    theme_font :heading_doctitle do
+      ink_prose title, align: :center # <2>
+    end
+    revremark = doc.attributes['revremark'] # <3>
+    if doc.author or doc.revdate or revremark # <4>
+      theme_font :base do
+        author_date_separator = (doc.author and doc.revdate) ? " – " : "" # <5>
+        revremark_separator = ((doc.author or doc.revdate) and revremark) ? " | " : "" # <6>
+        ink_prose "#{doc.author}#{author_date_separator}#{doc.revdate}#{revremark_separator}#{revremark}", align: :center # <7>
+      end
+    end
+    theme_margin :heading_doctitle, :bottom
```

Comment:
I've taken this line from https://github.com/asciidoctor/asciidoctor-pdf/issues/988#issuecomment-1128579996 and don't really know what it does. I _assume_ it adds some vertical space between the title (or here, the whole head section including the author/date/revremark line) above and the abstract or main content below, but I haven't verified that, yet.